Assets/Scripts/Character/PlayerHealth.cs

```diff
@@ -0,0 +1,49 @@
+using UnityEngine;
+
+public class PlayerHealthSystem : MonoBehaviour
```

Comment:
It looks like HealthSystem.cs and PlayerHealth.cs do the same thing. I think it should be abstracted into a single class that can be put on players and enemies.